File: app/src/main/java/com/example/nammapustaka/MainActivity.java
package com.example.nammapustaka;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    Button addBookBtn, viewBookBtn, scanBtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        addBookBtn = findViewById(R.id.addBookBtn);
        viewBookBtn = findViewById(R.id.viewBookBtn);
        scanBtn = findViewById(R.id.scanBtn);

        addBookBtn.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, AddBookActivity.class);
            startActivity(intent);
        });

        viewBookBtn.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, BookListActivity.class);
            startActivity(intent);
        });

        scanBtn.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, ScannerActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/example/nammapustaka/MainActivity.kt
package com.example.nammapustaka

import android.content.Intent
import android.os.Bundle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import com.example.nammapustaka.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.addBookBtn.setOnClickListener { startActivity(Intent(this, AddBookActivity::class.java)) }
        binding.viewBookBtn.setOnClickListener { startActivity(Intent(this, BookListActivity::class.java)) }
        binding.scanBtn.setOnClickListener { startActivity(Intent(this, ScannerActivity::class.java)) }
        binding.leaderboardBtn.setOnClickListener { startActivity(Intent(this, LeaderboardActivity::class.java)) }
        binding.reviewBtn.setOnClickListener { startActivity(Intent(this, ReviewActivity::class.java)) }
    }
}